### PR TITLE
add pun

### PR DESCRIPTION
**Pull request by kamaroolkarim**

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -265,5 +265,9 @@
   {
     "pun": "Q: What search algorithm did the wise men use to find the baby Jesus?",
     "punchline": "A: A*"
+  },
+  {
+    "pun": "Q: Why was the JavaScript developer sad?",
+    "punchline": "A: Because he didn't Node how to Express himself!"
```
Comment (AlexLakatos) [inline puns.json:271]: Hey @kamaroolkarim , thanks for the PR.

I'm trying to keep the puns gender-neutral, could you change it from `he` to `they`? Same for `himself` to `themselves`.

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
Gender-neutral puns.

File: puns.json (hunk 1 above)
Comment (kamaroolkarim) [reply puns.json:271]: ok. let me fix this

**Comment by kamaroolkarim** [comment]
alright. done

**Comment by AlexLakatos** [comment]
Thanks! And Happy Hacktoberfest!

**Pull request merged**